en/config/actuators.md

```diff
@@ -313,6 +313,51 @@ Configuring the Camera Capture / Trigger input requires a reboot to take effect
 You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
 These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
+### Output Functions
+
+Output functions are used to map the "logical functions" of an airframe, such as `Motor 1` or `Landing gear`, to physical outputs like FMU output pin 2.
+This makes it easy to use a particular output pin for almost any purpose.
+
+Some functions are only relevant to particular frames or output types, and will not be offered on others.
+
+Functions include: 
+
+- `Disabled`: Output has no assigned function.
+- `Constant_Min`: Output set to constant minimum value.
+- `Constant_Max`: Output is set to constant maximum value.
+- `Motor 1` to `Motor 12`: Output is indicated motor.
+  Only motors allowed for airframe are displayed.
+- `Servo 1` to `Servo 8`: Servo output.
+   These are further assigned a specific meaning based on airframe, such as "tilt servo", "left aileron".
+- `Offboard Acutator Set 1` to `Offboard Acutator Set 6`: MAVLink Payload output.
+- `Landing Gear`: Output is landing gear.
```

Comment:
Is there anything special you have to do for landing gear? (other than set this and wire PWM output?). We have no other instructions for using landing gear - would be good to create some - who's the expert?

Reply:
No, it just has 3 [action options](https://github.com/PX4/PX4-Autopilot/blob/main/msg/landing_gear.msg) and isn't that different from simple actuator. I think for 'instruction', it would just be:

1. Push the slider to maximum and verify that gear is up.
2. Push the slider to minimum and verify that gear is down
3. Adjust the 'reverse range' option appropriately, if the landing gear moves in the wrong direction

Reply:
I'm going to leave this part for now. If you know of an issue # for the problem though we could link it?

Reply:
> Oh actually it seems like it isn't even implemented properly

How so?

Reply:
FYI The main reason I want to leave this is that I need a lot more info than this about landing gear. But no more than this for this document.

Reply:
Changed my mind. Added landing gear/parachute sections at end. They will be OK, even if not exhaustive.